server/schemas/typeDefs.js

```diff
@@ -0,0 +1,66 @@
+const { gql } = require('apollo-server-express');
+
+/*
+
+User - it can be provider / consumer
+Parking place is master table for Parking Detail. as its info rarely get change
+Invetory represents providers entry for parking availabilty it can be durarion or one day.
+Reservation represents when Parking taken by Consumer and did payment process.
+
+*/ 
+const typeDefs = gql`
+
+type User {
+  _id: ID
+  firstName: String
+  lastName: String
+  email: String
+  phone:Int
+}
+
+type ParkingPlaces {
+  _id: ID
+  apt: String
+  street: String
+  city: String
+  state: String
+  zip:String
```

Comment:
could we externalize these properties into Address schema? We'd need it later for processing stripe transaction for a user